contracts/aggregator/fantom/flashloan/main.sol

```diff
@@ -0,0 +1,183 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+/**
+ * @title Flashloan.
+ * @dev Flashloan aggregator for Fantom.
+ */
+import "./helpers.sol";
+import "hardhat/console.sol";
```

Comment:
Remove this import and logs from from all places

Reply:
Done